modules/cli/apply/apply.go

```diff
@@ -187,16 +187,16 @@ func save(client *client.Getter, bundle *fleet.Bundle, imageScans ...*fleet.Imag
 		logrus.Infof("created: %s/%s\n", bundle.Namespace, bundle.Name)
 	} else if err != nil {
 		return err
+	} else {
```

Comment:
@nickgerace I actually found out this. We can't update outside of if block cause it will fail on the first run. Obj doesn't exist so when updating it it will just give error says `resource name may not be empty`.